docs/sections/getting_started/basics.md

```diff
@@ -353,106 +344,112 @@ Exit codes:
             310:  The output file could not be read.
             320:  The output file contains invalid output.
             410:  The sum of the operands is a negative number.
-
 ```
 
-There are several important things we can learn from the obtained output.
 The first is description of the calculation, which explains that it adds two numbers together.
-Then there are the inputs, of which 3 are required: the code to be used to add the numbers up, and the two numbers to be added up.
-Finally, among the outputs, you can see the `sum` which contains the result of the addition.
+Then there are the inputs, of which 3 are required: the `code` to be used to add the numbers up, and the two numbers (`x` and `y`) to add.
+Finally, note the `sum` among the outputs, which contains the result of the addition.
 
-Now that we understand what our `CalcJob` does and what it needs, let us see what we need to do to actually execute it.
+Now that we understand what our `CalcJob` does and what it needs, let's see what we need to do to run it.
```

Comment:
Again, nothing particularly for or against this, but in other reviews I was suggested to use the fuller versions ("let us", "will not", etc.) instead of the shorter informal ones ("Let's", "won't", etc). Just as a reminder in case we need to check the consistency of this.

Reply:
I prefer to be more informal, for sure. "Let us" sounds a little archaic to me. ^^